doc/releases/reef.rst

```diff
@@ -0,0 +1,281 @@
+====
+Reef
```

Comment:
We should include a major changes from quincy section, like we usually do https://docs.ceph.com/en/latest/releases/quincy/#major-changes-from-pacific

Reply:
@neha-ojha: I'll need the leads to tell me which are the major changes.

Reply:
Perhaps the rocksdb version bullet point (currently under RADOS) should go here.

Reply:
@zdover23 disregard my comment, we still want the rocksdb bullet under RADOS.